src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+
+public class FigureSupplier {
+    ColorSupplier colorSupplier = new ColorSupplier();
+    Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int figureRandom = random.nextInt(5);
```

Comment:
All magic numbers in your code should be constants. In this case, '5' should be a named constant to represent the count of figure types.